src/views/v2/month/calendar-body/day/calendar-events/calendar-event/featured-image.php

```diff
@@ -26,17 +26,17 @@
 ?>
 <div class="tribe-events-calendar-month__calendar-event-featured-image-wrapper">
 	<a
-		href="#"
-		title="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
+		href="<?php echo esc_url( $event->permalink ); ?>"
+		title="<?php echo esc_attr( $event->title ); ?>"
 		rel="bookmark"
 		class="tribe-events-calendar-month__calendar-event-featured-image-link"
 	>
 		<div class="tribe-events-calendar-month__calendar-event-featured-image tribe-common-c-image tribe-common-c-image--bg">
 			<div
 				class="tribe-common-c-image__bg"
-				style="background-image: url('<?php echo esc_attr( $event->thumbnail->full->url ); ?>');"
+				style="background-image: url('<?php echo esc_url( $event->thumbnail->full->url ); ?>');"
 				role="img"
-				aria-label="alt text here"
+				aria-label="<?php echo esc_attr( $event->title ); ?>"
```

Comment:
As above; here I'd keep using `get_the_title( $event->ID )` to get the filtered event title.